base/deprecated.jl

```diff
@@ -296,22 +296,8 @@ const MemoryError = OutOfMemoryError
 @deprecate filter!(r::Regex, d::Dict) filter!((k,v)->ismatch(r,k), d)
 
 # 1470
-@deprecate integer(s::AbstractString)   parseint(Int,s)
-@deprecate unsigned(s::AbstractString)  parseint(UInt,s)
-@deprecate int(s::AbstractString)       parseint(Int,s)
-@deprecate uint(s::AbstractString)      parseint(UInt,s)
-@deprecate int8(s::AbstractString)      parseint(Int8,s)
-@deprecate uint8(s::AbstractString)     parseint(UInt8,s)
-@deprecate int16(s::AbstractString)     parseint(Int16,s)
-@deprecate uint16(s::AbstractString)    parseint(UInt16,s)
-@deprecate int32(s::AbstractString)     parseint(Int32,s)
-@deprecate uint32(s::AbstractString)    parseint(UInt32,s)
-@deprecate int64(s::AbstractString)     parseint(Int64,s)
-@deprecate uint64(s::AbstractString)    parseint(UInt64,s)
-@deprecate int128(s::AbstractString)    parseint(Int128,s)
-@deprecate uint128(s::AbstractString)   parseint(UInt128,s)
-@deprecate float64(s::AbstractString)   parsefloat(Float64,s)
-@deprecate float32(s::AbstractString)   parsefloat(Float32,s)
```

Comment:
These are already handled by the generic method (https://github.com/JuliaLang/julia/blob/sb/parsefloat/base/deprecated.jl#L360-L377)